client/components/marketing-page-feature/index.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React, { ReactNode, FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	children: ReactNode;
+	description: string;
+	disclaimer?: string;
+	imageAlt?: string;
+	imagePath?: string;
+	title: string;
+}
+
+const MarketingPageFeature: FunctionComponent< Props > = ( {
+	children,
+	description,
+	disclaimer,
+	imageAlt,
+	imagePath,
+	title,
+} ) => {
+	return (
```

Comment:
fun fact! an arrow-function returns a value by default so we can skip the curlies `{` and the `return` statement 😄

Reply:
Ohhh yeah, cool beans.